config/sidekiq.yml

```diff
@@ -27,9 +27,6 @@
   ip_cleanup_scheduler:
     cron: '<%= Random.rand(0..59) %> <%= Random.rand(3..5) %> * * *'
     class: Scheduler::IpCleanupScheduler
-  preview_cards_cleanup_scheduler:
-    cron: '<%= Random.rand(0..59) %> <%= Random.rand(3..5) %> * * *'
-    class: Scheduler::PreviewCardsCleanupScheduler
```

Comment:
You should delete the `Scheduler::PreviewCardsCleanupScheduler` file too.

Reply:
I removed that file.